### ISSUE TITLE
Render alerts

### ISSUE THREAD

**Issue by toxin98** [shd101wyy/vscode-markdown-preview-enhanced#720]
I know `!!! note mainbody` work well, I don't know if it can be converted to github style.  

\> [!NOTE]  
\> Highlights information that users should take into account, even when skimming.

\> [!IMPORTANT]
\> Crucial information necessary for users to succeed.

\> [!WARNING]
\> Critical content demanding immediate user attention due to potential risks.

**Comment by asami0** [comment]
update?

**Comment by ericis** [comment]
Requirements are located here: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

**Comment by brandonpeebles** [comment]
This would be a great add 👍

**Comment by Waynting** [comment]
Yes — this is supported now. GitHub's alert syntax renders as callouts since crossnote [0.9.17](https://github.com/shd101wyy/crossnote/releases/tag/0.9.17) ([crossnote#387](https://github.com/shd101wyy/crossnote/pull/387) by @EmmetZ), shipped in extension [0.8.21](https://github.com/shd101wyy/vscode-markdown-preview-enhanced/releases/tag/0.8.21) (2026-03-15).

So all three of these now render, in the preview and in exports:

```markdown
> [!NOTE]
> Highlights information that users should take into account, even when skimming.

> [!IMPORTANT]
> Crucial information necessary for users to succeed.

> [!WARNING]
> Critical content demanding immediate user attention due to potential risks.
```

The `!!! note` admonition syntax you mentioned still works too — they are separate features, not a replacement.

@toxin98 this one looks closeable.
